apps/api/src/app/environments/usecases/regenerate-api-keys/regenerate-api-keys.usecase.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@nestjs/common';
+import { IApiKey, EnvironmentRepository } from '@novu/dal';
+import { ApiException } from '../../../shared/exceptions/api.exception';
+import { GetApiKeysCommand } from '../get-api-keys/get-api-keys.command';
+import * as hat from 'hat';
+
+@Injectable()
+export class RegenerateApiKeys {
+  constructor(private environmentRepository: EnvironmentRepository) {}
+
+  async execute(command: GetApiKeysCommand): Promise<IApiKey[]> {
+    const environment = await this.environmentRepository.findById(command.environmentId);
+
+    if (!environment) {
+      throw new ApiException(`Environment id: ${command.environmentId} not found`);
+    }
+
+    return await this.environmentRepository.updateApiKey(command.environmentId, hat(), command.userId);
```

Comment:
One small thing on my behalf to complete this PR. We have just merged a PR that handles the API key generation to enforce its uniqueness. So we would need you to use this use case for it:
https://github.com/novuhq/novu/blob/36327faeebd2135320dd823d97d251413670f3e4/apps/api/src/app/environments/usecases/generate-unique-api-key/generate-unique-api-key.usecase.ts#L8

Like it's been used here now:
https://github.com/novuhq/novu/blob/36327faeebd2135320dd823d97d251413670f3e4/apps/api/src/app/environments/usecases/create-environment/create-environment.usecase.ts#L21

I can do the change on your behalf if needed.

Reply:
Please if you don't mind, thank you

Reply:
Done. You would need to update your local branch as I had to force pushed with lease due the feature branch being behind of our main, so I could use the code we merged.